src/daft-local-execution/src/streaming_sink/anti_semi_hash_join_probe.rs

```diff
@@ -303,9 +301,9 @@ impl StreamingSink for AntiSemiProbeSink {
     #[instrument(skip_all, name = "AntiSemiProbeSink::finalize")]
     fn finalize(
         &self,
-        states: Vec<Box<dyn super::streaming_sink::StreamingSinkState>>,
+        states: Vec<Box<dyn super::base::StreamingSinkState>>,
         task_spawner: &ExecutionTaskSpawner,
-    ) -> super::streaming_sink::StreamingSinkFinalizeResult {
+    ) -> super::base::StreamingSinkFinalizeResult {
```

Comment:
BTW I moved the streaming sinks to a separate folder to make it clearer which are which